src/types/line.js

```diff
@@ -26,7 +26,7 @@ export default class LineAnnotation extends Element {
 	}
 
 	isOnLabel(x, y) {
-		const {labelRect} = this;
+		const labelRect = this.labelRect;
```

Comment:
This one also only adds weight.

Reply:
OKI! Done! I'm gonna push in next minutes.
I didn't know if there is a standard approach for that. I thought const {properties} = instance; could be used when you have more than 1 property to get, and not based on amount of bytes.

Reply:
I don't think there is a standard, except to try to have the files as small as possible. If we were using babel to support older browsers, then this one would be better, because the destructuring assignment is not supported by old browsers.